src/Models/EventAttendance.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace TruckersMP\APIClient\Models;
+
+use TruckersMP\APIClient\Collections\EventAttendeeCollection;
+
+class EventAttendance
+{
+    /**
+     * The confirmed attendee count.
+     *
+     * @var int
+     */
+    protected $confirmed;
+
+    /**
+     * The unsure attendee count.
+     *
+     * @var int
+     */
+    protected $unsure;
+
+    /**
+     * The confirmed users.
+     *
+     * @var EventAttendeeCollection|null
+     */
+    protected $confirmedUsers;
+
+    /**
+     * The unsure users.
+     *
+     * @var EventAttendeeCollection|null
+     */
+    protected $unsureUsers;
+
+    /**
+     * Create a new EventAttendance instance.
+     *
+     * @param  int  $confirmed
+     * @param  int  $unsure
+     * @param  array|null  $confirmedUsers
+     * @param  array|null  $unsureUsers
+     * @return void
+     */
+    public function __construct(
+        int $confirmed,
+        int $unsure,
+        ?array $confirmedUsers,
+        ?array $unsureUsers
+    ) {
+        $this->confirmed = $confirmed;
+        $this->unsure = $unsure;
+        if (isset($confirmedUsers)) {
+            $this->confirmedUsers = new EventAttendeeCollection($confirmedUsers);
+        }
+        if (isset($unsureUsers)) {
+            $this->unsureUsers = new EventAttendeeCollection($unsureUsers);
+        }
```

Comment:
These should be empty collections instead of `null`.

Reply:
I usually agree, but there was a reason behind this in this usecase.
When testing the API responses, it did not return the attendee arrays for a certain event on the VTC event endpoint.

If it sometimes can return no attendee arrays, returning an empty collection would make it in my opinion seem like there are no users marked as confirmed/unsure, instead of the data just not being in the response. Null seemed like a better option in this case, otherwise end users will think "oh, an empty collection. There must not be any sign-ups to this event".

However, I haven't been able to reproduce this possible bug/issue since I mentioned it to you yesterday evening, so if both the VTC and non-VTC endpoints both should *always* return attendee arrays, I'll remove the isset check.

Reply:
Actually, you are right they are not listed in certain API end-points.
Let's keep it like this then. :+1: